src/libpmem2/map.c

```diff
@@ -59,6 +59,7 @@ pmem2_get_length(const struct pmem2_config *cfg, size_t file_len,
 	if (ret)
 		return ret;
 
+	ASSERTne(alignment, 0);
```

Comment:
Assertions should be used only when the source of inconsistency is internal to the application/library.

In this case, alignment comes from the operating system and I don't see any code that handles this anywhere. We should handle it as an error inside of pmem2_config_get_alignment, document it, and then we can add an assert here.